app/move/controllers/assign/base.js

```diff
@@ -1,6 +1,9 @@
+const { pick } = require('lodash')
+
+const presenters = require('../../../../common/presenters')
 const CreateBaseController = require('../create/base')
 
-class CreateAllocationBaseController extends CreateBaseController {
+class AssignToAllocationBaseController extends CreateBaseController {
```

Comment:
Could we do the same with all these naming changes too please and remove the reference to the allocation. I'd either suggest `AssignPersonToMoveBaseController` or `AssignBaseController`. I think the second fits better with the existing naming convention but happy if you want to be more explicit.